src/screens/BookmarkCategoryScreen.js

```diff
@@ -83,6 +88,9 @@ export const BookmarkCategoryScreen = ({ navigation }) => {
 const styles = StyleSheet.create({
   icon: {
     paddingHorizontal: normalize(14)
+  },
+  warningText: {
+    paddingVertical: normalize(14)
```

Comment:
why is this added?
in my tests this adds padding in landscape mode, which i have not guessed when viewing the code.
so in portrait there is 14 padding top and obbtom and for landscape 28 each? is that intended?

Reply:
this is a repetition . warningText is already present in BookmarkScreen so here is not necessary ?